recommenders/models/newsrec/io/mind_iterator.py

```diff
@@ -119,7 +119,7 @@ def init_behaviors(self, behaviors_file):
 
                 history = [self.nid2index[i] for i in history.split()]
                 history = [0] * (self.his_size - len(history)) + history[
-                    : self.his_size
+                    -self.his_size :
```

Comment:
Hi @thaiminhpv, thanks for the contribution. It seems some files got mixed up in your PR. 

To fix it, please change the [PR to staging](https://github.com/recommenders-team/recommenders/blob/main/CONTRIBUTING.md), and [sign the commit](https://github.com/recommenders-team/recommenders/wiki/How-to-sign-commits).

Let me know if you have any difficulty or need any help

Reply:
@miguelgfierro Thank you, I've changed merge target branch to the staging branch. My commit was also signed. Please have a look at it.